City-of-Pori/pori-kada
# PORI-87: Color configuration by theme

Color configuration by theme.

To test:
1. drush updb -y && drush fra -y && drush cc all
2. Go to /admin/structure/taxonomy/theme and edit the terms
3. In the term settings there should be a list of Colors (red, orange, yellow, green, greendark). Choose a color for each term. (correct colors can be found below)
4. After setting the term colors, go to menus and edit main-navigation. For the top level links you can give in "Menu item attributes" a class called theme-color-green for example.
5. After setting colors for menu items, go to the site and confirm that the menu items you set colors to have that color as hover and active-trail etc.
6. Go to edit some news items that are lifted to main-news and set a theme for them, after that go to frontpage and confirm that the item has colored border according to that theme color.
7. Do the above to some of the content that is lifted to topics and confirm working color.

Theme colors according to the story description

- Asuminen ja ympäristö -> red
- Vapaa-aika ja liikunta -> orange
- Kasvatus ja koulutus -> yellow
- Sosiaali- ja terveys -> green
- Työ ja yrityspalvelut -> greenDark

NOTICE: If you have existing colors set for theme terms, you need to set those N/A for the colors to change properly.
